src/backend/utils/text-parser.js

```diff
@@ -1,11 +1,23 @@
-const jsdom = require('jsdom');
+const puppeteer = require('puppeteer');
 
-const { JSDOM } = jsdom;
-
-// Simple function to convert html to text
-// Issue #75: https://github.com/Seneca-CDOT/telescope/issues/75
+/**
+ * Simple function to convert html to text
+ * @returns {string} a string of text parse from html
+ * Issue #75: https://github.com/Seneca-CDOT/telescope/issues/75
+ */
 exports.run = async function(html) {
-  const frag = JSDOM.fragment(html);
-  const result = await frag.textContent;
-  return result;
+  let browser;
+  try {
+    browser = await puppeteer.launch();
+    const page = await browser.newPage();
+
+    await page.setContent(html, {
+      waitUntil: 'domcontentloaded',
+    });
+
+    const result = await page.evaluate('document.body.innerText');
+    return result;
+  } finally {
+    await browser.close();
```

Comment:
This is throwing on CircleCI, probably because Chrome isn't installed:

```
 TypeError: Cannot read property 'close' of undefined

      19 |     return result;
      20 |   } finally {
    > 21 |     await browser.close();
         |                   ^
      22 |   }
      23 | };
```

Let's change this line to:

```js
} finally {
  if(browser) {
    await browser.close();
}
```